task.ps1

```diff
@@ -40,3 +40,16 @@ New-AzVm `
 -SshKeyName $sshKeyName  -PublicIpAddressName $publicIpAddressName
 
 # ↓↓↓ Write your code here ↓↓↓
+$Params = @{
+    ResourceGroupName  = $resourceGroupName
+    VMName             = $vmName
+    Name               = 'CustomScript'
+    Publisher          = 'Microsoft.Azure.Extensions'
+    ExtensionType      = 'CustomScript'
+    TypeHandlerVersion = '2.1'
+    Settings          = @{
+      fileUris = @('https://raw.githubusercontent.com/p1tonidze/azure_task_12_deploy_app_with_vm_extention/develop/install-app.sh')
```

Comment:
Please update the URL in the `fileUris` parameter to reflect your own GitHub username instead of 'p1tonidze'. This change is essential for the script to download the correct installation script.